es/es-server/src/main/java/org/elasticsearch/index/translog/Translog.java

```diff
@@ -1178,24 +1178,22 @@ public long version() {
             return this.version;
         }
 
-        public VersionType versionType() {
-            return versionType;
-        }
-
         @Override
         public Source getSource() {
             return new Source(source, routing);
         }
 
         private void write(final StreamOutput out) throws IOException {
-            out.writeVInt(SERIALIZATION_FORMAT);
+            final int format = out.getVersion().onOrAfter(Version.V_4_0_0) ? SERIALIZATION_FORMAT : FORMAT_6_0;
```

Comment:
to confirm - we use our own version as the serialization format?

Reply:
Yep, there is no other Version anymore.